src/app/pages/wallets/my-wallets/wallet-form-modal/dialog-create-or-update-wallet/wallet-form-modal.component.html

```diff
@@ -0,0 +1,11 @@
+<form [formGroup]='form'>
+    <h1 mat-dialog-title>{{ data ? "Update Wallet" : "Create Wallet" }}</h1>
+    <mat-form-field appearance="fill">
+        <mat-label>Wallet Name</mat-label>
+        <input matInput [formControl]='$any(form).controls["name"]'  >
```

Comment:
You should not use `$any` here,, what you are missing are types for the form:
https://angular.io/guide/typed-forms
This is new feature introduced in Angular 14 :)